app/util/menus/EditMenu.ts

```diff
@@ -335,11 +336,13 @@ export class EditMenu extends Menu {
     this.append({
       label: $t('Create Stream Output Projector'),
       click: () => this.projectorService.createProjector(ERenderingMode.OBS_STREAMING_RENDERING),
+      enabled: this.streamingService.state.selectiveRecording || Utils.isDevMode(),
```

Comment:
Dev mode behavior is for Trevor to test, or?

Reply:
yep exactly